cmd/minikube/cmd/start.go

```diff
@@ -222,7 +222,7 @@ func provisionWithDriver(cmd *cobra.Command, ds registry.DriverState, existing *
 	k8sVersion := getKubernetesVersion(existing)
 	cc, n, err := generateClusterConfig(cmd, existing, k8sVersion, driverName)
 	if err != nil {
-		return node.Starter{}, errors.Wrap(err, "Failed to generate config")
```

Comment:
Instead of here we can catch during, validateFlags .

Reply:
https://github.com/kubernetes/minikube/blob/13895fa1daa7282d067da618f11c5dc2fcda2808/cmd/minikube/cmd/start.go#L819-L822

@medyagh If I implement the check here, I'll have to check the flag in a list of valid flags which can be cumbersome in my opinion. So, I think like the other attributes we should let this one fall onto the switch statement as well.

Reply:
no this is not the right way to do it, to let minikube go and fail then act on failure,
we have validateFlags and we can detect there before sending minikube to the field.

please handle the validation over there.